rpm/meson.spec

```diff
@@ -8,9 +8,8 @@ Source:         %{name}-%{version}.tar.bz2
 Patch0:         patch-macros.patch
 BuildArch:      noarch
 BuildRequires:  python3-devel >= 3.6.0
+BuildRequires:  ccache
```

Comment:
Can understand removing the Requires if building with meson works without it now, but why is this now pulled in for building meson itself?

Reply:
Heh, you are right. I didn't question the fact that it was needed, but looking at the build log etc it for sure isn't used.